contracts/MerkleAirdrop.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2;
+
+import "./MerkleProof.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/utils/structs/BitMaps.sol";
+
+/**
+ * @dev A contract to allow users to claim tokens via a 'merkle airdrop'.
+ */
+contract MerkleAirdrop is Ownable {
+    using BitMaps for BitMaps.BitMap;
+
+    address sender;
+    IERC20 public token;
+    bytes32 public merkleRoot;
+    BitMaps.BitMap private claimed;
+
+    event MerkleRootChanged(bytes32 merkleRoot);
+    event Claim(address indexed claimant, uint256 amount);
+
+    /**
+     * @dev Constructor.
+     * @param _sender The account to send airdrop tokens from.
+     * @param _token The token contract to send tokens with.
+     */
+    constructor(address _sender, IERC20 _token) {
```

Comment:
Do we need to pass through the token? Would hardcoding be one less thing to worry about?

Reply:
I think it's as easy to supply it as a parameter than it is to hardcode it in the source.